extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/FixedPageSizeColumnChunkPageStore.java

```diff
@@ -17,6 +17,10 @@
 import java.lang.ref.WeakReference;
 import java.util.Arrays;
 
+/**
+ * Unused class, use {@link OffsetIndexBasedColumnChunkPageStore} instead because to check if the page sizes are fixed
+ * without actually reading the page headers, we need offset index.
+ */
 class FixedPageSizeColumnChunkPageStore<ATTR extends Any> extends ColumnChunkPageStore<ATTR> {
```

Comment:
Should I delete this class? I also hit exceptions while using it but didn't dig too deep. So it has some unidentified bugs too.

Reply:
Yes, delete it. I understand that that's a feature in the offset-indexed version.

Reply:
Yes, I would delete if unused.